_data/sites.json

```
@@ -15715,6 +15715,16 @@
         ]
     },
 
+    {
+        "name": "Shotgun.live",
+        "url": "https://support.shotgun.live/hc/en-us/articles/360016912580--I-want-to-delete-my-account-",
+        "difficulty": "hard",
```

```suggestion
        "difficulty": "medium",
```
I'd say medium, because even though one has to download a mobile app, it still can be done without having to reach out for their support team.

---

Fair enough!